internal/pkg/cli/app_show.go

```diff
@@ -36,25 +36,32 @@ type showAppVars struct {
 type showAppOpts struct {
 	showAppVars
 
-	store            store
-	w                io.Writer
-	sel              appSelector
-	pipelineSvc      pipelineGetter
-	newVersionGetter func(string) (versionGetter, error)
+	store             store
+	w                 io.Writer
+	sel               appSelector
+	pipelineSvc       pipelineGetter
+	deployStoreClient selector.DeployStoreClient
+	newVersionGetter  func(string) (versionGetter, error)
 }
 
 func newShowAppOpts(vars showAppVars) (*showAppOpts, error) {
-	defaultSession, err := sessions.ImmutableProvider(sessions.UserAgentExtras("app show")).Default()
+	sessProvider := sessions.ImmutableProvider(sessions.UserAgentExtras("app show"))
+	defaultSession, err := sessProvider.Default()
 	if err != nil {
 		return nil, fmt.Errorf("default session: %w", err)
 	}
 	store := config.NewSSMStore(identity.New(defaultSession), ssm.New(defaultSession), aws.StringValue(defaultSession.Config.Region))
+	deployStoreClient, err := deploy.NewStore(sessProvider, store)
+	if err != nil {
+		return nil, fmt.Errorf("deploy store client: %w", err)
```

Comment:
nit: to be consistent on error message wrapping:
```suggestion
		return nil, fmt.Errorf("connect to deploy store: %w", err)
```